src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/ExternalUserSearchController.kt

```diff
@@ -15,13 +15,13 @@ import org.springframework.web.bind.annotation.RequestParam
 import org.springframework.web.bind.annotation.ResponseStatus
 import org.springframework.web.bind.annotation.RestController
 import uk.gov.justice.digital.hmpps.manageusersapi.config.ErrorResponse
-import uk.gov.justice.digital.hmpps.manageusersapi.service.UserSearchService
```

Comment:
Suggestion: is it a good idea to add into packages like : 'nomis' , 'external' under service and resources

Reply:
Yeah - that was the alternative - Happy either way tbh....I'll refactor like that - cheers.